demisto_sdk/commands/validate/validators/PB_validators/PB108_is_valid_task_id.py

```diff
@@ -0,0 +1,70 @@
+from __future__ import annotations
+
+from typing import Iterable, List
+
+from demisto_sdk.commands.common.tools import is_string_uuid
+from demisto_sdk.commands.content_graph.objects.base_playbook import TaskConfig
+from demisto_sdk.commands.content_graph.objects.playbook import Playbook
+from demisto_sdk.commands.validate.validators.base_validator import (
+    BaseValidator,
+    ValidationResult,
+)
+
+ContentTypes = Playbook
+
+
+class IsValidTaskIdValidator(BaseValidator[ContentTypes]):
+    error_code = "PB108"
+    description = "Validate that the task ID and the 'id' under the 'task' field are from UUID format."
+    rationale = "Validate that the task ID and the 'id' under the 'task' field are from UUID format."
```

Comment:
It shouldn't be the same. 
The rationale here is that each task should have a unique id to avoid unknown behavior and breaking the playbook